pkg/flowaggregator/flowaggregator.go

```diff
@@ -152,13 +173,18 @@ func (fag *flowAggregator) Run(stopCh <-chan struct{}) {
 	}
 	fag.aggregationProcess = ap
 
-	exportTicker := time.NewTicker(fag.exportFrequency)
+	exportTicker := time.NewTicker(fag.exportInterval)
 	defer exportTicker.Stop()
 	go fag.collectingProcess.Start()
 	go fag.aggregationProcess.Start()
 	for {
 		select {
 		case <-stopCh:
+			fag.collectingProcess.Stop()
+			fag.aggregationProcess.Stop()
+			if fag.exportingProcess != nil {
+				fag.exportingProcess.CloseConnToCollector()
+			}
 			break
```

Comment:
we should use return here. Break only breaks from select.

Reply:
Thanks, addressed.